### PR TITLE
Update screenshots and fix old wording from OData  example

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 1]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
-The APIkit OData Extension builds the services from the entity data model (EDM), which OData services use to formalize their resource descriptions. You write the EDM in RAML using EDM-compatible data types to represent API entities. +
-The APIkit OData Extension generates Mule flows to handle OData requests. After configuring a data source and adding endpoints to the flows, deploying the app exposes a RESTful API and an OData API accessible through different URLs. The OData MySQL example on this page shows how to use an HTTP-based data service to query a MySQL database.
+The APIkit OData Extension supports OData (Open Data Protocol) to create HTTP-based data services.
 
-You can use the APIkit OData Extension to expose a legacy API as an OData API, orchestrate data before exposing it in Salesforce, or create a bidirectional OData API for Oracle, DB2, or Azure DB, for example.
+The APIkit OData Extension builds the services from the entity data model (EDM), which OData services use to formalize their resource descriptions. You can write the EDM in RAML using EDM-compatible data types to represent API entities.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc:9]: APIkit OData extension? (I'm not sure)

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 2]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+
+The APIkit OData extension generates Mule flows to handle OData requests. You can use the APIkit OData Extension to, for example, expose a legacy API as an OData API, orchestrate data before exposing it in Salesforce, or create a bidirectional OData API for Oracle, DB2, or Azure DB.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]: Here it is APIkit OData extension (I think this is right)

Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
The APIkit OData extension generates Mule flows to manage OData requests. You can use the APIkit OData extension to, for example, expose a legacy API as an OData API, orchestrate data before exposing it in Salesforce, or create a bidirectional OData API for Oracle, DB2, or Azure DB.
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 3]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+
+The OData MySQL example on this page shows how to use an HTTP-based data service to query a MySQL database.
 
 == Prerequisites
 
-The following software is required for creating and using an OData-enabled API with APIkit:
+The following software is required to create and using an OData REST API with APIkit:
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
The following software is required to create and use an OData REST API with APIkit:
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 4]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
 * Anypoint Studio 7.1.4 and later
-* Maven
 
-== Installing the APIKit OData Extension
+Additionally, you must download the link:{attachmentsdir}/odata.raml[OData example entity data model].
+
+== Install the APIKit OData Extension
 
-You install the APIKit OData Extension from Anypoint Studio as described in the following procedure:
+To install the APIKit OData Extension in Anypoint Studio:
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
To install the APIKit OData extension in Anypoint Studio:
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 5]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
 
-. From the *Help* menu in Studio, select *Install New Software*.
-. Click *Add...*
+. Select *Help* > *Install New Software*.
+. Select *Add...*
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Click *Add...*
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 6]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
++
+image::add-update-site.png[]
 . In the *Name* field, type `APIkit for ODATA Update Site`.
 . In the *Location* field, type `+http://studio.mulesoft.org/s4/apikit-for-odata/+`.
-. Click *OK*.
-+
-Studio displays a list of items to select.
+. Select *Add*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Click *Add*.
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 7]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
-. Click *Next*.
-+
-An Install Remediation Page might show.
-+
-. Click *Next*, and click *Next* again to install the remedial software.
-. Accept the terms and conditions of the product, then click *Finish*.
-+
-An Installing Update (%) message appears at the bottom right corner of Studio.
+image::add-repository-odata.png[]
+. Select the *Anypoint ODATA Plugin* box, and select *Next*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Select the *Anypoint ODATA Plugin* box, and click *Next*.
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 8]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
 +
+image::select-odata-extension.png[]
+. Accept the terms and conditions of the product, then select *Finish*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Accept the terms and conditions of the product, then click *Finish*.
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 9]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
 . Restart Studio to complete the installation.
 
-== Using the APIkit OData Extension
+== Implement an API Using the APIkit OData Extension
 
-After installing the prerequisite software, perform the following high-level steps to use the APIkit OData Extension:
+The following example shows you how to implement an OData enabled REST API:
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
The following example shows you how to implement an OData-enabled REST API:
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 10]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
-
-+
-. Create a new Mule project in Anypoint Studio. Click *File*, then select *New > Mule Project*.
-+
-Select the *Specify API definition file location or URL* on the New Mule Project dialog.
-. Click *Finish*
-. Copy the `odata.raml` to /src/main/resources/api in Studio project explorer.
+. In Studio, select *File* > *New* > *Mule Project*.
+. In the *Project Name* field, type the name of your Mule project.
+. Select *Finish*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Click *Finish*.
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 11]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+. Copy the `odata.raml` file you downloaded in the prerequisites section to the `/src/main/resources/api` directory in your project.
 . In the project explorer, right-click `odata.raml` and select *Mule* > *Generate OData API from RAML Types*.
 +
-The OData Extension generates the `api.raml` in /src/main/resources/api and `ODataLibrary.raml` in /src/main/resources/api/libraries. Generated flows are displayed on the canvas when you select `api.xml`.
-+
-image::creating-an-odata-api.png[]
+image::generate-odata.png[]
+. The OData Extension generates the necessary files for youe OData API imlementation:
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. The OData extension generates the necessary files for your OData API implementation:
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 12]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
-
-* Locally
-+
-Right-click your project, then select *Run As* > *Mule Application*.
-* To CloudHub
-+
-Right-click your project, then select *Anypoint Platform* > *Deploy to Cloud*.
-
-Once deployed, you can also register the OData app in external OData consumer services like Salesforce.
+<1> The Mule configuration file with the flows for the API implementation.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
<1> The Mule configuration file with the flows for the API implementation
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 13]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+<1> The generated RAML definition.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
<1> The generated RAML definition
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 14]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+<1> The generated RAML libraries.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
<1> The generated RAML libraries
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 15]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
-. Select *Anypoint Studio* > *Packaged mule application (.jar)*. Click *Next*.
-+
-The Import Mule Project dialog shows.
-+
-. Browse for the jar file downloaded and select it. Click *Finish*.
-+
-[TIP]
-The test-drive project shows in the project explorer. APIkit OData Extension generates flows that are displayed in the canvas when you click `api.xml` in src/main/app in the project explorer.
+. Install a MySQL database and launch MySQL.
+. Download MySQL project example link:{attachmentsdir}/apikit-odata-example-master.jar[apikit-odata-example-master.jar].
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Download the MySQL project example link:{attachmentsdir}/apikit-odata-example-master.jar[apikit-odata-example-master.jar].
```

File: modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc [hunk 16]
```diff
@@ -4,113 +4,81 @@ include::_attributes.adoc[]
...
+. Run the `example.sql` script located in `src/main/resources` using MySQL commands to load data into a database named `apikit-odata-example`.
 
-+
-. In Studio, edit the `mule-configuration.properties` file inside the src/main/resources folder, and set the following properties to access the local MySQL database:
+=== Implement the OData API
+
+. In Studio, Select *File* > *Import*.
+. Select *Anypoint Studio* > *Packaged mule application (.jar)* > *Next*.
+. Browse for the `apikit-odata-example-master.jar` file and select *Finish*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc]:
```suggestion
. Browse for the `apikit-odata-example-master.jar` file and click *Finish*.
```

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**